migrations/3.0.57

```diff
@@ -0,0 +1,22 @@
+-- recreate beacons to reflect the new radius
+
+local migrated_storage = {}
+
+for _, unit_number in pairs(storage.vatbrains) do
+    local beacon = unit_number.beacon
+    local vatbrain = unit_number.vatbrain
+
+    if not beacon then return end
```

Comment:
this will terminate the entire loop on finding an entry without a beacon

Reply:
Also need a validity check here:
<img width="419" height="245" alt="{5FF37FF6-FA29-4BE7-B2DA-7E39D08F13C7}" src="https://github.com/user-attachments/assets/cfb5728c-584a-4c22-ab6d-dec87741295e" />

e.g.
```lua
    if not beacon or not beacon.valid or not vatbrain or not vatbrain.valid then
        goto continue
     end
 ```
 and then add the continue marker directly before the end of the loop
 ```lua
    migrated_storage[new_beacon.unit_number] = {beacon = new_beacon, vatbrain = vatbrain}
    ::continue::
end
 ```
 
 Could also just put all of the code inside a positive valid check